ports/espressif/background.c

```diff
@@ -40,14 +40,17 @@
 #include "common-hal/pulseio/PulseIn.h"
 #endif
 
-void port_background_task(void) {
+void port_background_tick(void) {
```

Comment:
Is this going to run a lot more often because it's on tick rather than task? Is that OK?

Reply:
It's the reverse, `task` is more often than `tick`. The old task naming originally threw me off because I didn't know it depended on tick enable.

Reply:
Aha, thanks!